ceph-nvmeof.conf

```diff
@@ -18,6 +18,10 @@ state_update_interval_sec = 5
 #min_controller_id = 1
 #max_controller_id = 65519
 enable_discovery_controller = false
+#omap_file_lock_duration = 60
```

Comment:
Hello @gbregman
Is it possible to eliminate the exposed configuration for `omap_file_lock_duration`? My perspective is that lock duration should not be a user-exposed parameter. Instead, requests for lock_exclusive should inherently imply a 'duration' of 'infinity'. I'd appreciate your thoughts on this.

Reply:
@baum I wanted to have this exposed so in case of some disaster in the field we could easily disable the lock by setting this value to 0.